ReachabilityUI/ReachabilityUI/Shared/ReachabilityRepository.swift

```diff
@@ -70,10 +88,13 @@ final class ReachabilityManager: ReachabilityRepository {
     private func start() {
         // Checks if we are already listening
         guard !isListening else { return }
```

Comment:
just a comment, but this I think I would improve in some way...its just hard to read I think

Reply:
Like, improve the "isListening" part? :)